chainer/link.py

```diff
@@ -142,6 +142,10 @@ def __init__(self, **params):
             shape, dtype = _ensure_shape_dtype(value)
             self.add_param(name, shape, dtype=dtype)
 
+    def __call__(self, *args, **kwargs):
+        out = self.forward(*args, **kwargs)
```

Comment:
How about one liner code?

Reply:
forward_preprocess and forward_postprocess will be inserted before and after this line. So I just want to leave it as is.

Reply:
>forward_preprocess and forward_postprocess will be inserted before and after this line. So I just want to leave it as is.

That can be done in that PR.
Each PR should be self-consistent.

Reply:
Why isn't this self-consistent? I don't think it breaks "self-consistency".

Reply:
>forward_preprocess and forward_postprocess will be inserted before and after this line. So I just want to leave it as is.

Doesn't this mean it's imperfect until forward_preprocess and forward_postprocess will be inserted?

Reply:
OK, I accept current code.